docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+    sciencebeam-utils:
+        build:
+            context: .
+        image: elifesciences/sciencebeam-utils:${IMAGE_TAG}
+        environment:
+            - TWINE_USERNAME=$TWINE_USERNAME
```

Comment:
I'm assuming the credentials are coming from environment variables

Reply:
This is how they work now on the `elife-libraries` server: https://github.com/elifesciences/builder-base-formula/blob/e1d3be3643c5c014d71802ad9f45ffbf77ae01b7/elife/pypi.sls

Reply:
Okay, shall mount `~/.pypirc` instead or in addition.